core/src/main/java/org/apache/accumulo/core/file/rfile/bcfile/BoundedRangeFileInputStream.java

```diff
@@ -95,18 +92,7 @@ public int read(final byte[] b, final int off, int len) throws IOException {
         throw new IOException("Stream closed");
       }
       in.seek(pos);
-      try {
-        ret = AccessController.doPrivileged(new PrivilegedExceptionAction<Integer>() {
```

Comment:
I removed this in master branch with ACCUMULO-4498, and think it's safe to remove, but also think it could cause confusion for anybody who might have worked out a custom security policy for their environment. I think that's probably extremely low risk, though. I think it's fine to remove.